google_guest_agent/main.go

```diff
@@ -247,6 +251,38 @@ func runCmdOutput(cmd *exec.Cmd) *execResult {
 	return &execResult{code: 0, out: stdout.String()}
 }
 
+func runCmdOutputAndTimeout(cmd *exec.Cmd, timeoutSec int) *execResult {
```

Comment:
i think this can instead be a function that wraps `runCmdOutput`, but passes in an `exec.CommandContext` instead. it will avoid the need to replicate so much code. see: http://golang.org/pkg/os/exec/#CommandContext

Reply:
there haven't been any pushes since this was resolved, so unresolving

Reply:
Weird, I might be losing my mind, try now!